From 5acd546473f5b2816b19f70e8e760cf6b5ac39e0 Mon Sep 17 00:00:00 2001
From: Wolfgang Grandegger <wg@aries-embedded.de>
Date: Fri, 27 Mar 2020 11:29:45 +0100
Subject: [PATCH 4/5] usb: dwc2: add device-tree property to use an external
 VBUS supply

Signed-off-by: Wolfgang Grandegger <wg@aries-embedded.de>
---
 drivers/usb/dwc2/params.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/usb/dwc2/params.c b/drivers/usb/dwc2/params.c
index 38e895692c2d..a1dc1cde7380 100644
--- a/drivers/usb/dwc2/params.c
+++ b/drivers/usb/dwc2/params.c
@@ -477,6 +477,9 @@ static void dwc2_get_device_properties(struct dwc2_hsotg *hsotg)
 
 	if (of_find_property(hsotg->dev->of_node, "disable-over-current", NULL))
 		p->oc_disable = true;
+
+	if (of_find_property(hsotg->dev->of_node, "external-vbus-supply", NULL))
+		p->phy_ulpi_ext_vbus = true;
 }
 
 static void dwc2_check_param_otg_cap(struct dwc2_hsotg *hsotg)
-- 
2.34.1

